docs/content/foundations/spacing-units/padding-margin.mdx

```diff
@@ -1,8 +1,9 @@
 ---
 browser_title: NCI Design System Padding and Margin
+page_title: Padding and Margin
 nav_label: Padding and Margin
 nav_order: 20
-template_type: default
+template_type: utility
 ---
 
 # Margin and Padding
```

Comment:
Please remove this as the page_title now displays. (p.s. can you please change the page_title and nav_label to be Margin and Padding.